tests/feature/providers/test-testable-provider.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace Alley\WP\WP_Page_Cache_Control\Tests\Feature\Providers;
+
+use Alley\WP\WP_Page_Cache_Control\Providers\Testable_Provider;
+use Alley\WP\WP_Page_Cache_Control\Tests\Test_Case;
+
+class Test_Testable_Provider extends Test_Case {
+	protected Testable_Provider $provider;
+
+	protected function setUp(): void {
+		parent::setUp();
+
+		add_filter( 'wp_page_cache_control_provider', fn () => Testable_Provider::class );
+
+		$this->assertInstanceOf( Testable_Provider::class, wp_page_cache_control() );
+
+		$this->provider = wp_page_cache_control();
+	}
+
+	public function test_ttl() {
+		wp_page_cache_control()->ttl( 1800 );
+
+		$this->assertTrue( $this->provider->has_ttl( 1800 ) );
+		$this->assertFalse( $this->provider->has_ttl( 123 ) );
+	}
+
+	public function test_disable_cache() {
+		wp_page_cache_control()->disable_cache();
```

Comment:
Question about how this is called throughout tihs file... since above we declare `$this->provider = wp_page_cache_control();` should we be using `$this->provider` subsequently? Maybe doesn't matter...

Reply:
I went with `$this->provider` to make it cleaner